src/start.js

```diff
@@ -127,4 +127,26 @@ export function start() {
       // logger.log();
     });
 
+  program.command('update')
+    .alias('u')
+    .description('Update aurealia')
+    .option('--nuclear', "Go nuclear")
+
+    .action(cli.execute('updater'))
+
+    .on('--help', function () {
+      utils.example('update', {
+        nuclear: {
+            flags: '--nuclear <name>'
+          , info : 'Blow it up'
+          , required: false
+        },
+      });
+
+      // logger.log('  Examples:');
+      // logger.log();
+      // logger.log('    $ aurelia plugin -a i18n');
+      // logger.log('    $ aurelia p -r i18n -c');
+      // logger.log();
```

Comment:
Good question, I had that there uncommented before but I noticed that all of the examples were commented above in the same file.  Actually while reading it those are from the plugin anyway so need to remove.